src/analysis/retail/monk/windwalker/modules/apl/AplCheck.tsx

```diff
@@ -118,22 +151,188 @@ export const apl = build([
   },
 ]);
 
-export const aplProps = (events: AnyEvent[], info: PlayerInfo): AplRuleProps => {
-  const check = aplCheck(apl);
-  return {
-    apl: apl,
-    checkResults: check(events, info),
-  };
+export const conduit_apl = build([
+  {
+    spell: TALENTS.STRIKE_OF_THE_WINDLORD_TALENT,
+    condition: and(
+      hasChi(2),
+      buffMissing(SPELLS.HEART_OF_THE_JADE_SERPENT_UNITY),
+      spellCooldownRemaining(TALENTS.FISTS_OF_FURY_TALENT, { atMost: 6000 }),
+    ),
+  },
+  {
+    spell: TALENTS.CELESTIAL_CONDUIT_TALENT,
+    condition: describe(
+      and(
+        buffRemaining(SPELLS.HEART_OF_THE_JADE_SERPENT_BUFF, 10, { atMost: 1000 }, true),
+        spellCooldownRemaining(TALENTS.FISTS_OF_FURY_TALENT, { atMost: 6000 }),
+      ),
+      (tense) => (
+        <>
+          <SpellLink spell={SPELLS.HEART_OF_THE_JADE_SERPENT_BUFF} /> is missing
+        </>
+      ),
+    ),
+  },
+  {
+    spell: TALENTS.RISING_SUN_KICK_TALENT,
+    condition: and(hasChi(2), buffPresent(TALENTS.XUENS_BATTLEGEAR_TALENT)),
+  },
+  {
+    spell: SPELLS.TIGER_PALM,
+    condition: describe(
+      and(
+        hasResource(RESOURCE_TYPES.CHI, { atMost: hasTalent(TALENTS.ASCENSION_TALENT) ? 4 : 3 }),
+        hasResource(RESOURCE_TYPES.ENERGY, {
+          // over 85% of max energy
+          atLeast:
+            (BASE_ENERGY_MAX +
+              (hasTalent(TALENTS.ASCENSION_TALENT) ? ASCENSION_ENERGY_MAX_ADDITION : 0) +
+              (hasTalent(TALENTS.INNER_PEACE_TALENT) ? INNER_PEACE_ENERGY_MAX_ADDITION : 0)) *
+            0.85,
+        }),
```

Comment:
same note here about `hasTalent` usage. these cases both complex enough (in terms of how many options there are) and simple enough (only controlling a threshold for a `hasResource` check) that using the function approach seems better to me